src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -238,15 +240,20 @@ private void createInfoPanel() {
         TitledBorder tb = new TitledBorder(new LineBorder(Color.darkGray),
                                            I18nUtils.getString("EXAM_ADDRESS_1"));
         infoPanel_.setBorder(tb);
+       
         infoPanel_.setToolTipText(I18nUtils.getString("EXAM_DISPLAY_2"));
+        String unUsedString = "Hello World ";
 
         JLabel nameLabel = new JLabel(I18nUtils.getString("EXAM_NAME_3"));
+        
         JLabel streetLabel = new JLabel(I18nUtils.getString("EXAM_STREET_4"));
         JLabel apartmentLabel = new JLabel(I18nUtils.getString("EXAM_APARTME_5"));
+
         JLabel cityLabel = new JLabel(I18nUtils.getString("EXAM_CITY_6"));
         JLabel zipLabel = new JLabel(I18nUtils.getString("EXAM_ZIPCODE_7"));
-        JLabel phoneLabel = new JLabel("Phone:");
-        JLabel emailLabel = new JLabel("Email:");
+
+        JLabel phoneLabel = new JLabel("Phone: ");
+        JLabel emailLabel = new JLabel("Email: ");
 
 
         JLabel street = new JLabel(street_);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "street" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)